geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/stats/StatsBasedEstimator.scala

```diff
@@ -242,7 +244,9 @@ class CountEstimator(sft: SimpleFeatureType, stats: GeoMesaStats) extends LazyLo
         bounds.bounds.map(_.bounds)
       }
       extractedBounds.flatMap { bounds =>
-        if (bounds.contains((None, None))) {
+        if (bounds.isEmpty) {
+          Some(0L) // disjoint range
+        } else if (bounds.exists(_ == (None, None))) {
```

Comment:
actually, i think `(None, None)` is appropriate here - the bounds class encapsulates 2 optional endpoints, to represent open ended bounds; e.g. `foo < 5` -> `(None, Some(5))`. So `(None, None)` represents open ended bounds on both sides, i.e. an inclusive filter.